Dockerfile

```diff
@@ -21,9 +21,17 @@ RUN apk add --no-cache $PACKAGES && \
 # Final image
 FROM alpine:edge
 
+ENV GAIA /gaia
+
 # Install ca-certificates
 RUN apk add --update ca-certificates
-WORKDIR /root
+
+RUN addgroup gaiauser && \
```

Comment:
Why not just `gaia` instead of `gaiauser`? Not a big deal anyway - it's just conventional on UNIX systems to name daemon's system users after the daemon itself.

Reply:
will make the change.